README.md

```diff
@@ -60,7 +60,11 @@ A set of buildConfigs for building custom Jenkins images for OpenShift.
 ## Related Content
 
 * [Container Pipelines](https://github.com/redhat-cop/container-pipelines) - A set of Jenkins piplines for OpenShift
-* [Labs CI/CD](https://github.com/rht-labs/labs-ci-cd) - A comprehensive end to end pipeline using many modern testing and quality tools
+* [Labs Ubiquitous Journey](https://github.com/rht-labs/ubiquitous-journey) - A collection of ArgoCD apps for:
```

Comment:
Could you add something like a footnote, etc. here around `f.k.a. Labs CI/CD`?

Reply:
@oybed  - Do you want me to just keep Labs CI/CD in there and mention it as a good place to turn to for OCP v.3.X?

Reply:
@springdo Sure, that sounds good.

Reply:
Cool added both for the history books 
📖 👴